### PR TITLE
fix(cmd/influx): show see help, instead of usage

### PR DESCRIPTION
**Pull request by kelwang**
Closes https://github.com/influxdata/influxdb/issues/13768

> so the plan for this one:
> 
> * Do not show the "help" message for any errors for any command
> * Add a message to the end of each error saying `See 'influx <cmd> -h' for help`

Manually tested

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [x] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: cmd/influx/main.go [hunk 1]
```diff
@@ -72,15 +73,44 @@ func out(w io.Writer) genericCLIOptfn {
...
+	var err error
+	var cmd *cobra.Command
+
+	// Workaround FAIL with "go test -v" or "cobra.test -test.v", see #155
+	if args == nil && filepath.Base(os.Args[0]) != "cobra.test" {
+		args = os.Args[1:]
+	}
+
+	if parent.TraverseChildren {
+		cmd, _, err = parent.Traverse(args)
```
Comment (jsteenb2) [inline cmd/influx/main.go]: since they're being ignored, can you just do `cmd, _, _ =` for these spots?

File: go.mod [hunk 2]
```diff
@@ -75,10 +74,10 @@ require (
 	github.com/prometheus/common v0.6.0
 	github.com/prometheus/procfs v0.0.3 // indirect
 	github.com/satori/go.uuid v1.2.0
-	github.com/spf13/cast v1.2.0
-	github.com/spf13/cobra v0.0.3
+	github.com/spf13/cast v1.3.0
+	github.com/spf13/cobra v0.0.5
```
Comment (jsteenb2) [inline go.mod]: hmm this roles cobra and vipers versions, what's the driver behind that?

**Comment by kelwang** [comment]
> lgtm, question about needing to import the newer versions of things is a potential blocker though

Not a blocker, but they have a lot of cleanup, should only help

File: go.mod (hunk 2 above)
Comment (kelwang) [reply go.mod]: i only updated the cobra version, viper is picked up by go mod vendor and go mod tidy

**Pull request merged**